lasagne/tests/layers/test_recurrent.py

```diff
@@ -1,12 +1,13 @@
 import pytest
+
 from lasagne.layers import RecurrentLayer, LSTMLayer, CustomRecurrentLayer
-from lasagne.layers import InputLayer, DenseLayer, GRULayer, Gate
+from lasagne.layers import InputLayer, DenseLayer, GRULayer, Gate, Layer
 from lasagne.layers import helper
 import theano
 import theano.tensor as T
 import numpy as np
 import lasagne
-
+from mock import Mock
```

Comment:
You removed a newline here. pep8 is offended! (It requires two empty lines in front of every function definition, including the first one.)